src/css/styles.scss

```diff
@@ -247,6 +248,13 @@ main {
   width: 250px;
 }
 
+.tags {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows: 1fr;
+  width: 60%;
+}
+
```

Comment:
I think even with the two columns, overflow scroll might still be a nice fix

Reply:
I agree!